test/game-test.js

```diff
@@ -44,7 +44,6 @@ describe('Game', function() {
 
   it('should display puzzle', () => {
     domUpdates.startGame(game);
```

Comment:
This could also be getting replaced by the spy you're setting on line 13. 

Also, your class structure may be a little cleaner if you move the invocation of `domUpdates.startGame` into `game.startGame`. This keeps your Game class in control of the logic of things, and leaves domUpdates to handle the visual side of things